spec/spec_helper.rb

```diff
@@ -17,7 +19,12 @@
 
 Capybara.app = Chitter
 
+
 RSpec.configure do |config|
+  config.before(:each) do
+    setup_test_database
+  end 
+
   config.after(:suite) do
     puts
     puts "\e[33mHave you considered running rubocop? It will help you improve your code!\e[0m"
```

Comment:
Overall, nice effort but as I can understand its work in progress :) I would try to keep Views and Controllar separate to make it easier to follow.